finance-api/api-reference.md

```diff
@@ -0,0 +1,335 @@
+</details>
+
+
+## Fetch payout information
+Returns payout related information from a merchant's liquid account.
+
+A payout is to deposit the account balance to the merchant's bank account or their PayPal Wallet for PayPal users at the [scheduled time](https://www.zettle.com/help/articles/1084784-deposits). The scheduled time varies depending on the country.
+
+If the merchant's configuration has a minimum account balance, then the payout will deposit the account balance minus the minimum account balance.
+
+```
+GET /organizations/{organizationUuid}/payout-info
+```
+
+See example [Fetch payout information on a specific period](#fetch-payout-information-on-a-specific-day).
+
+### Parameters
+
+<details open="true">
+<summary>Click to hide request parameters.</summary>
+
+|Name |Type |In |Required/Optional |Description
+|:---- |:---- |:---- |:---- |:----
+|organizationUuid |string |path |required |Unique identifier for your organisation. You can specify the value with one of the following: <br/><ul><li> Use `self` as the value. This will retrieve your organizationUuid from the authentication token in the request.</li><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li></ul> 
+|at |string |query |optional |Used to fetch payouts at a certain point in UTC time. If it's used, any transaction after that time will be ignored. If it's not used, the balance of all transactions at the current point of time is returned. <br/>You can specify a UTC time in one of the following formats: <br/><ul><li>`YYYY-MM` to specify a month. For example, `2020-11`. By default, it specifies the first second of the first day in the month.</li><li>`YYYY-MM-DD` to specify a date. For example, `2020-11-29`.</li><li>`YYYY-MM-DDThh:mm:ss` to specify a time. For example, `2020-11-29T03:10:02`.</li></ul>
+</details>
+
+
+### Responses
+<details open="true">
+<summary>Click to hide HTTP status codes.</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the operation is successful.  
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request.
+|401 Unauthorized |Returned when one of the following occurs: <br/><ul><li> The authentication information is missing in the request.</li><li>The authentication token has expired.</li><li>The authentication token is invalid.</li></ul>
+|403 Forbidden |Returned when the scope being used in the request is incorrect.  
+</details>
+
+<details open="true">
+<summary>Click to hide response attributes.</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the upcoming payout. 
+|totalBalance |integer |The account balance in the currency's smallest unit. For example, 300 with currency GBP is £3. It can be negative, such as when refunds are greater than sales.
```

Comment:
Here you mention that 300 with currency GBP is £3, but in the previous API you don't. 
Shouldn't you mention there as well?